R/reportWaterUsage.R

```diff
@@ -0,0 +1,43 @@
+#' @title reportWaterUsage
+#' @description reports water usage for agricultural sector, crops and livestock
+#' 
+#' @export
+#' 
+#' @param gdx GDX file
+#' @param detail logical. Setting to FALSE reports for agricultural sector, TRUE reports for combined, crops and livestock separately
+#' @return water usage as MAgPIE object Unit: see names
+#' @author Florian Humpenoeder, Vartika Singh, Miodrag Stevanovic
+#' @examples
+#' 
+#'   \dontrun{
+#'     x <- reportWaterUsage(gdx)
+#'   }
+#' 
+
+
+reportWaterUsage<-function(gdx, detail=TRUE) {
+  
+  x <- water_usage(gdx,level="regglo",users=NULL,sum=TRUE,digits=3)[,,"agriculture"]
+  getNames(x) <- "Resources|Water|Withdrawal|Agriculture (km3/yr)"
+  return(x)
+  
+  if (detail==TRUE) {
+    out <-x 
```

Comment:
you don't need this assignment here. Instead just use `x` object at line 29. `out` is not necessary